src/lib/parser/games/vanilla/config.ts

```diff
@@ -67,45 +217,48 @@ export class VanillaConfig implements GameConfig {
-    partyPokemonSize: 104,
+    partyStartOffset: 0x238, // Actual party data offset in SaveBlock1 (0x234 is party count)
+    partyPokemonSize: 100, // sizeof(struct Pokemon) = 100 bytes in vanilla Emerald  
     maxPartySize: 6,
     pokemonNicknameLength: 10,
     pokemonTrainerNameLength: 7,
-    playTimeHours: 0x0E, // Different from Quetzal
-    playTimeMinutes: 0x0F, // Different from Quetzal
-    playTimeSeconds: 0x10, // Different from Quetzal
+    playTimeHours: 0x0E, // u16 playTimeHours in SaveBlock2
+    playTimeMinutes: 0x10, // u8 playTimeMinutes in SaveBlock2  
+    playTimeSeconds: 0x11, // u8 playTimeSeconds in SaveBlock2
     pokemonData: {
-      personality: 0x00,
-      otId: 0x04,
-      nickname: 0x08,
-      otName: 0x14,
-      currentHp: 0x23,
-      species: 0x28,
-      item: 0x2A,
-      move1: 0x34,
-      move2: 0x36,
-      move3: 0x38,
-      move4: 0x3A,
-      pp1: 0x3C,
-      pp2: 0x3D,
-      pp3: 0x3E,
-      pp4: 0x3F,
-      hpEV: 0x40,
-      atkEV: 0x41,
-      defEV: 0x42,
-      speEV: 0x43,
-      spaEV: 0x44,
-      spdEV: 0x45,
-      ivData: 0x50,
-      status: 0x57,
-      level: 0x58,
-      maxHp: 0x5A,
-      attack: 0x5C,
-      defense: 0x5E,
-      speed: 0x60,
-      spAttack: 0x62,
-      spDefense: 0x64,
+      // BoxPokemon part (first 80 bytes)
+      personality: 0x00,     // u32 personality
+      otId: 0x04,           // u32 otId 
+      nickname: 0x08,       // u8 nickname[10] 
+      otName: 0x14,         // u8 otName[7] (offset 20 decimal)
+      // Species and other data are encrypted in substructs - need different approach
+      species: 0x20,        // In encrypted substruct - placeholder for now
+      item: 0x22,           // In encrypted substruct - placeholder for now
+      move1: 0x24,          // In encrypted substruct - placeholder for now
+      move2: 0x26,          // In encrypted substruct - placeholder for now  
+      move3: 0x28,          // In encrypted substruct - placeholder for now
+      move4: 0x2A,          // In encrypted substruct - placeholder for now
+      pp1: 0x2C,            // In encrypted substruct - placeholder for now
+      pp2: 0x2D,            // In encrypted substruct - placeholder for now
+      pp3: 0x2E,            // In encrypted substruct - placeholder for now
+      pp4: 0x2F,            // In encrypted substruct - placeholder for now
+      hpEV: 0x30,           // In encrypted substruct - placeholder for now
+      atkEV: 0x31,          // In encrypted substruct - placeholder for now
+      defEV: 0x32,          // In encrypted substruct - placeholder for now  
+      speEV: 0x33,          // In encrypted substruct - placeholder for now
+      spaEV: 0x34,          // In encrypted substruct - placeholder for now
+      spdEV: 0x35,          // In encrypted substruct - placeholder for now
+      ivData: 0x36,         // In encrypted substruct - placeholder for now
+      // Pokemon struct part (last 20 bytes)
```

Comment:
we want to decrypt the substructs correctly. we want to find the correct iv/ev and use the calculateTotalStatsDirect function to confirm our findings to match the total stats